project/ScoverageSettings.scala

```diff
@@ -22,7 +22,7 @@ object ScoverageSettings {
   def apply(): Seq[Def.Setting[_ >: String with Double with Boolean]] =
```

Comment:
Minor point but I saw Pauls note this morning, maybe we should be switching off coverage until it supports scala 3?

Reply:
Yeah, good point.. scoverage has dropped significantly.